### PR TITLE
chore: publish docker @next tag in master builds and @latest in git tags

### PR DESCRIPTION
**Pull request by menduz**
Adapt catalyst to follow the conventions of other deployments with a canary release schema `@next` and stable releases `@latest`. Setting us up for a continuous release schema in the master branch to test in Decentraland's dev environment.

The rules of tagging are:

- Publish in `@latest` on semver (X.X.X) git tags
- Publish in `@next` every main or master branch builds.

This PR only modifies the docker pipelines. Not the npm pipelines.

### CODE REVIEW

**Review by nchamo — COMMENTED** [review]
Hey @menduz , this looks great!

I have a question though. Are we loosing the ability to release a custom version on a work in progress branch?

Because right now we sometimes release a custom image before merging to master, to test the new code

**Comment by menduz** [comment]
Mmh you are right. Do not merge this. I forgot that part. It was in my notes and the playbooks for versioning. Will handle it wirh rc tags like in npm packages.

**Review by nchamo — APPROVED** [review]
Great work!

**Pull request merged**
